.github/workflows/CI.yml

```diff
@@ -218,20 +213,68 @@ jobs:
             -DSUNSHINE_CONFIGURE_ONLY=ON \
             ..
 
+      - name: Checkout Flatpak Builder Tools
+        uses: actions/checkout@v3
+        with:
+          repository: flatpak/flatpak-builder-tools
+          path: flatpak-builder-tools
+
+      - name: Generate cache for npm
+        run: |
+          npm install --lockfile-version 2 --package-lock-only
+          pip install ./flatpak-builder-tools/node
+          flatpak-node-generator -o npm-generated-sources.json npm package-lock.json
+          mkdir -p npm
+          cp npm-generated-sources.json build/npm-generated-sources.json
+          mv npm-generated-sources.json npm/npm-generated-sources.json
+          mv package-lock.json npm/package-lock.json
+
+      - name: Checkout Flathub Shared Modules
+        uses: actions/checkout@v3
+        with:
+          repository: flathub/shared-modules
+          path: build/shared-modules
+
+      - name: External Data Checker
+        run: |
+          flatpak run org.flathub.flatpak-external-data-checker build/dev.lizardbyte.sunshine.yml
+
       - name: Build Linux Flatpak
         working-directory: build
         run: |
-          sudo su $(whoami) -c 'flatpak run org.flatpak.Builder --arch=${{ matrix.arch }} --repo=repo --force-clean \
-            --stop-at=cuda build-sunshine dev.lizardbyte.sunshine.yml'
+          mkdir -p ~/.local/bin
+          wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O ~/.local/bin/yq
+          chmod +x ~/.local/bin/yq
+          # Replace value for filesystem permission
+          yq -i '.modules[-2].build-options.build-args[0] = "--filesystem=${{ github.workspace }}"' dev.lizardbyte.sunshine.yml
+          # Replace value for WORKSPACE variable
+          yq -i '.modules[-2].build-options.env.WORKSPACE = "${{ github.workspace }}"' dev.lizardbyte.sunshine.yml
```

Comment:
This yq tool seems pretty cool, but I have a concern/question with replacing values like this.

Currently we're replacing values using cmake and the configure command. There are already 3 values replaced just for flatpak. I'm thinking it should be done in a similar way for these as to not complicate an offline build.

Can just modify this step above...
```bash
          cmake -DGITHUB_CLONE_URL=${clone_url} \
            -DGITHUB_BRANCH=${branch} \
            -DGITHUB_COMMIT=${commit} \
            -DSUNSHINE_CONFIGURE_FLATPAK_MAN=ON \
            -DSUNSHINE_CONFIGURE_ONLY=ON \
            ..
```